### PR TITLE
Requirements updates for gitc 66

### PR DESCRIPTION
**Pull request by zacheryschiller**
##### Description

These pip requirements updates are to ensure that we can build cleanly from scratch with the two updates from dependabot.

##### Refers/Fixes

 This is required for https://gitcoin.atlassian.net/browse/GITC-66.

##### Testing

Tested locally.

### CODE REVIEW

**Comment by zacheryschiller** [comment]
We should squash when we merge.

**Comment by thelostone-mc** [comment]
Looks like the merge didn't happen right!
Could you cherry-pick the commit and update this branch ?

https://docs.gitcoin.co/mk_contributors/#step-4-commit

File: requirements/base.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ Markdown==3.2.1
 reportlab==3.5.6
 requests
 requests_oauthlib
-Pillow==4.0.0
+Pillow==8.2.0
```
Comment (thelostone-mc) [inline requirements/base.txt:21]: Question: did we test the pillow version bump ? I wasn't able to test this out locally which is why I was hesitating bumping this up

Comment (zacheryschiller) [reply requirements/base.txt:21]: I tested this on the "staging" that I deployed with ansible.

**Review by chibie — APPROVED** [review]

**Comment by thelostone-mc** [comment]
Closing as this is still in draft! Do reopen if this is ready toget in

**Pull request closed without merging**